WGLMakie/assets/sprites.vert

```diff
@@ -78,17 +78,17 @@ void main(){
     // get_position() returns the (relative) position of the current quad vertex
 
     vec3 f32c_scale = get_f32c_scale();
-    vec2 markersize = f32c_scale.xy * get_markersize();
+    vec2 ms = f32c_scale.xy * get_quad_scale();
     vec2 quad_offset = f32c_scale.xy * get_quad_offset();
     vec3 marker_offset = f32c_scale * tovec3(get_marker_offset());
 
-    vec2 bbox_radius = 0.5 * markersize;
+    vec2 bbox_radius = 0.5 * ms;
     vec2 sprite_bbox_centre = quad_offset + bbox_radius;
-    f_sprite_scale = markersize;
+    f_sprite_scale = ms;
     mat4 pview = projection * view;
-    mat4 trans = get_transform_marker() ? model : mat4(1.0);
+    mat4 trans = get_transform_marker() ? model_f32c : mat4(1.0);
```

Comment:
I think this is wrong because trans applies to marker_offset which is independent of f32convert